slack_sdk/socket_mode/aiohttp/__init__.py

```diff
@@ -0,0 +1,193 @@
+import asyncio
+import logging
+from asyncio import Future
+from asyncio import Queue
+from logging import Logger
+from typing import Union, Optional, List, Callable, Awaitable
+
+import aiohttp
+from aiohttp import ClientWebSocketResponse, WSMessage, WSMsgType, ClientConnectionError
+
+from slack_sdk.socket_mode.async_client import AsyncBaseSocketModeClient
+from slack_sdk.socket_mode.async_listeners import (
+    AsyncWebSocketMessageListener,
+    AsyncSocketModeRequestListener,
+)
+from slack_sdk.socket_mode.request import SocketModeRequest
+from slack_sdk.web.async_client import AsyncWebClient
+
+
+class SocketModeClient(AsyncBaseSocketModeClient):
+    logger: Logger
+    web_client: AsyncWebClient
+    app_token: str
+    wss_uri: Optional[str]
+    auto_reconnect_enabled: bool
+    web_socket_message_queue: Queue
+    web_socket_message_listeners: List[
+        Union[
+            AsyncWebSocketMessageListener,
+            Callable[
+                ["AsyncBaseSocketModeClient", dict, Optional[str]], Awaitable[None]
+            ],
+        ]
+    ]
+    socket_mode_request_listeners: List[
+        Union[
+            AsyncSocketModeRequestListener,
+            Callable[["AsyncBaseSocketModeClient", SocketModeRequest], Awaitable[None]],
+        ]
+    ]
+
+    web_socket_message_receiver: Optional[Future]
+    web_socket_message_processor: Future
+
+    proxy: Optional[str]
+    ping_interval: float
+    current_session: Optional[ClientWebSocketResponse]
+    current_session_monitor: Optional[Future]
+
+    on_message_listeners: List[Callable[[WSMessage], None]]
+    on_error_listeners: List[Callable[[WSMessage], None]]
+    on_close_listeners: List[Callable[[WSMessage], None]]
+
+    def __init__(
+        self,
+        app_token: str,
```

Comment:
To make it specific, I suggest using this name over `token` - what do you think? @stevengill